openshift-container-platform-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -448,4 +694,817 @@
+    - key: b
+      text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-19 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-19 (2)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-19 (3)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-19 (4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-20
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-21
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        A control response to SA-21(a) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-460
+    - key: b
+      text: |
+        A control response to SA-21(b) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-460
+
+- control_key: SA-21 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        A control response to SA-21 is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-464
+
+- control_key: SA-22
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        A control response to SA-22(a) is planned. Progress
+        can be tracked at:
+
+        https://issues.redhat.com/browse/CMP-465
+    - key: b
+      text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
+
+- control_key: SA-22 (1)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
```

Comment:
To preemptively answer questions about what versions of OCP are currently supported, what do you think about elaborating this answer to include a link to https://access.redhat.com/support/policy/updates/openshift ?

Reply:
@shawndwells what does this have to do with SA-22(1) Validation of Screening?

Reply:
SA-22(1) | UNSUPPORTED SYSTEM COMPONENTS \| ALTERNATIVE SOURCES FOR CONTINUED SUPPORTThe organization provides [Selection (one or more): in-house support; [Assignment: organization-defined support from external providers]] for unsupported information system components.
-- | --

Reply:
@redhatrises SA-22(1) is not related to "Validation of Screening"

Reply:
ps - resolving conversations hides them, and requires manually going through 1500+ lines to see where the comment was made. Please wait until the conversation is done before resolving.

Reply:
@shawndwells I dyslexia'd it.

Since SA-22(1) is about ALTERNATIVE SOURCES FOR CONTINUED SUPPORT for unsupported information system components, trying to understand why we would add the Red Hat support page for OCP versions? What if I am using a component that was never integrated into OCP?

Reply:
@shawndwells added the lifecycle link to SA-22(a) is it is where we originally identified where to put the link to the lifecycle.

Reply:
Much better fit. Good call!